src/Compilers/VisualBasic/Portable/BoundTree/Statement.vb

```diff
@@ -1487,9 +1489,11 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Implements IVariableDeclarationStatement
 
             Private ReadOnly _variables As ImmutableArray(Of IVariableDeclaration)
+            Private ReadOnly _syntax As SyntaxNode
 
-            Public Sub New(variables As ImmutableArray(Of IVariableDeclaration))
+            Public Sub New(variables As ImmutableArray(Of IVariableDeclaration), syntax As SyntaxNode)
```

Comment:
Thanks. Then it seems `_syntax` can also use the specific type. And the cast above may become irrelevant.

---
In reply to: [119499384](https://github.com/dotnet/roslyn/pull/19885#discussion_r119499384) [](ancestors = 119499384)